packages/amplify-category-storage/src/provider-utils/awscloudformation/import/import-dynamodb.ts

```diff
@@ -348,14 +350,14 @@ export const importedDynamoDBEnvInit = async (
 
       // Copy over the required input arguments to currentEnvSpecificParameters
       /* eslint-disable no-param-reassign */
-      currentEnvSpecificParameters.tableName = sourceEnvParams.tableName;
-      currentEnvSpecificParameters.region = sourceEnvParams.region;
-      currentEnvSpecificParameters.arn = sourceEnvParams.arn;
-      currentEnvSpecificParameters.streamArn = sourceEnvParams.streamArn;
-      currentEnvSpecificParameters.partitionKeyName = sourceEnvParams.partitionKeyName;
-      currentEnvSpecificParameters.partitionKeyType = sourceEnvParams.partitionKeyType;
-      currentEnvSpecificParameters.sortKeyName = sourceEnvParams.sortKeyName;
-      currentEnvSpecificParameters.sortKeyType = sourceEnvParams.sortKeyType;
+      currentEnvSpecificParameters.tableName = resourceParamManager.getParam('tableName')!;
+      currentEnvSpecificParameters.region = resourceParamManager.getParam('region')!;
```

Comment:
```suggestion
      currentEnvSpecificParameters.region = resourceParamManager.getParam(DDBParams.region)!;
```
like an enu

Reply:
are you saying create an enum for all of these parameters?

Reply:
yeah like for strings we are directly using  (table name and region), just a nit